pkg/apis/extensions/v1beta1/types.go

```diff
@@ -34,7 +34,7 @@ type ScaleStatus struct {
 	Replicas int32 `json:"replicas"`
 
 	// label query over pods that should match the replicas count. More info: http://releases.k8s.io/HEAD/docs/user-guide/labels.md#label-selectors
-	Selector map[string]string `json:"selector,omitempty"`
+	Selector *LabelSelector `json:"selector,omitempty"`
```

Comment:
I would like this to be a string (in the same format as the query-param syntax), if that's ok with the auto-scaling folks, so that it can be passed back to the API server without introspection.